### PR TITLE
Update about.md

### PR DESCRIPTION
**Pull request by tahabi**
Still needs a lot of improvement

see #158

### CODE REVIEW

File: ocfweb/docs/docs/about.md [hunk 1]
```diff
@@ -32,19 +33,22 @@ resources available, the OCF also provides a variety of educational services.
...
+organization, the [Computer Science Undergraduate Association][csua].
 
 Policies for this cluster and decisions for the OCF organization are made by
-the OCF Board of Directors, a group of students elected each semester by the
-OCF membership.  The Board meetings, held at least once every two weeks when
-classes are in session, are open to the public and provide a chance for any
-user to express their concerns or opinions about the OCF. [Minutes of previous
-meetings][minutes] are also available online.
+the OCF Board of Directors, an open body composed of OCF members who are interested
+in participating in the OCF decision-making process. The Board convenes at General
```
Comment (kpengboy) [inline ocfweb/docs/docs/about.md]: BoD convenes at BoD meetings. General meetings don't exist anymore.

Comment (tahabi) [inline ocfweb/docs/docs/about.md]: thanks

File: ocfweb/docs/docs/about.md [hunk 2]
```diff
@@ -32,19 +33,22 @@ resources available, the OCF also provides a variety of educational services.
...
-well as online sources of information about computing on the Berkeley campus
-and how to use the available resources.  Additionally, the OCF has conducted
-career fairs in the past with its sister organization, the [Computer Science
-Undergraduate Association][csua].
+community.  We also used to host the UNIX SysAdmin [DeCal courses][decal]. Now,
+we host regular Tech Talks that explore new technologies and how they might
+apply to elements of the OCF infrastructure. We also offer e-mail and in-person
+consulting with our volunteer staff members, as well as online sources of information
+about computing on the Berkeley campus and how to use the available resources.
+Additionally, the OCF has conducted career fairs in the past with its sister
```
Comment (NickImpicciche) [inline ocfweb/docs/docs/about.md]: group past information about career fair and decals together, don't split it up with tech talks in between

File: ocfweb/docs/docs/about.md [hunk 3]
```diff
@@ -30,21 +31,24 @@ the ground floor of MLK Student Union.
...
 
 Policies for this cluster and decisions for the OCF organization are made by
-the OCF Board of Directors, a group of students elected each semester by the
-OCF membership.  The Board meetings, held at least once every two weeks when
-classes are in session, are open to the public and provide a chance for any
-user to express their concerns or opinions about the OCF. [Minutes of previous
-meetings][minutes] are also available online.
+the OCF Board of Directors, an open body composed of OCF members who are
+interested in participating in the decision-making process. The Board
+convenes at BoD Meetings, held at least every week when classes are in session,
```
Comment (NickImpicciche) [inline ocfweb/docs/docs/about.md]: I mean, they're every week. We would never have two in a week. When this every other week comment was written bod meetings were less frequent and consistent.

Comment (tahabi) [inline ocfweb/docs/docs/about.md]: Yeah, I removed the every other week thing and replaced it with "at least every week." Are you saying we should remove "at least"?

Comment (NickImpicciche) [inline ocfweb/docs/docs/about.md]: Yeah p much. It sounds kind of weird, and I mean I guess technically we sometimes cancel bod, but I just think saying it's every week is fine.

**Comment by matthew-mcallister** [comment]
I appreciate the update, but this page still needs to be totally rewritten some day.

Also, consider squashing some of those trivial commits and renaming others so it's clear what they refer to. I'm not uber picky about this, but remember that these go in the commit history after the PR is merged.

**Comment by tahabi** [comment]
Yeah, it probably does need to be rewritten,

**Pull request merged**